src/app/shared/search/search-result.component.ts

```diff
@@ -164,6 +166,11 @@ export class SearchResultComponent implements OnChanges {
     let isAscending = true;
     let isDescending = true;
 
+    if (this.comparator(column) === undefined) {
+      console.error('Error: The case_field_type ' + column.case_field_type.type + ' cannot be sorted on ' +
```

Comment:
can we log this as a `warning` rather than an error? Also, can we change the message? e.g.
console.warn('Sort error: unknown sort comparator for ' + column.case_field_type.type)

Reply:
Done.